build.gradle

```diff
@@ -1,79 +1,9 @@
 plugins {
-	id 'java'
-	id 'org.springframework.boot' version '3.2.5'
-	id 'io.spring.dependency-management' version '1.1.4'
-	id 'jacoco'
-	id 'net.researchgate.release' version '3.0.2'
-	id 'checkstyle'
+	id 'uk.gov.laa.ccms.springboot.laa-ccms-spring-boot-gradle-plugin' version '0.0.1-d85b520-SNAPSHOT'
```

Comment:
Should probably make this 0.0.1 once the PR is merged for the common repo

Reply:
Yep I'll be updating all the snapshot versions before merging.